modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -10330,6 +10330,20 @@ export const console: ConsoleNS = {
                             message: "User added successfully"
                         }
                     },
+                    addUserPendingApproval: {
+                        error: {
+                            description: "{{description}}",
```

Comment:
Shall we add a description here.

Reply:
All the other notification types have it this way. So not changing it.